### PR TITLE
Add QtCreator quick start guide

### PR DESCRIPTION
**Pull request by m-kuhn**
@rduivenvoorde This is another piece in the puzzle to create a good developer guide. Not sure how to best structure this. (From a toctree view).

### CODE REVIEW

**Comment by NathanW2** [comment]
Very nice!  I will add comments for anything extra I find

File: source/site/getinvolved/development/qtcreator.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Next you will probably want to use the full power of your machine when
+compiling. for that switch to the tab "Projects" on the left side of QtCreator.
+Locate the section ``Build Steps``, click Details and write ``-j8`` where you
+can replace 8 with as many cores as you would like to assign to the build
+process.
+
+.. note:: **Dealing with CMake problems**
+          Make sure that you have installed all build requirements, there are
+          some good hints in the [install
+          file](https://htmlpreview.github.io/?https://github.com/qgis/QGIS/blob/master/doc/INSTALL.html)
```
Comment (DelazJ) [inline source/site/getinvolved/development/qtcreator.rst]: I'm not sure that's the right way to add link in rst. IMO rather use `install file <https://htmlpreview.github.io/?https://github.com/qgis/QGIS/blob/master/doc/INSTALL.html>`_

Comment (m-kuhn) [inline source/site/getinvolved/development/qtcreator.rst]: Changed, thank you

Comment (DelazJ) [inline source/site/getinvolved/development/qtcreator.rst]: Sorry, seems that the comment formating has shifted my **`** character at the beginning and the end.

Comment (m-kuhn) [inline source/site/getinvolved/development/qtcreator.rst]: better now?

**Comment by m-kuhn** [comment]
Ah, right. Should be ok now and I realized that there's even a preview.

Thank you @DelazJ

**Comment by m-kuhn** [comment]
@NathanW2 There's even more to review now :)

File: source/site/getinvolved/development/git.rst [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  added
+
+To check which remotes are currently setup, you may use the following command:
+
+``git remote -v``
+
+Normally the first remote which gets added gets the name ``origin``. I renamed
+it here (in the file ``.git/config``) to avoid confusion and give explicit names.
+
+.. code::
```
Comment (m-kuhn) [inline source/site/getinvolved/development/git.rst:86]: @rduivenvoorde how can I make a code block in webpage rst style?

Comment (DelazJ) [inline source/site/getinvolved/development/git.rst:86]: Looking at https://github.com/qgis/QGIS-Website/edit/master/source/site/getinvolved/development/qgisdevelopersguide.rst#L115, I'd say that you just miss the empty line after the double colons.

Comment (m-kuhn) [inline source/site/getinvolved/development/git.rst:86]: Thanks!

File: source/site/getinvolved/development/git.rst [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+  will be much easier for him to follow your thoughts.
+* When somebody in the future wants to know why a particular line of code was
+  changed he can look, when this was changed and if there's a verbose message
+  he will be able to stay in line with your plans from now.
+
+You can now change more and add more commits. Once you are happy with the
+changes, you can propse to integrate them back into the master branch. First
+you will need to push your changes to github.
+
+.. code::
```
Comment (DelazJ) [inline source/site/getinvolved/development/git.rst:154]: miss an empty line

Comment (m-kuhn) [inline source/site/getinvolved/development/git.rst:154]: Changed, thanks

**Comment by m-kuhn** [comment]
Can somebody check if this is properly linked in the TOC?
Once that's done I think it is safe to merge it and help for more collaboration from other devs.

File: source/site/getinvolved/development/index.rst [hunk 4]
```diff
@@ -335,3 +335,4 @@ Contributors of new functions are encouraged to let people know about their cont
     pythonplugindevelopment
     addinggrasstools
     qgisdevelopersguide
+    qtcreator.rst
```
Comment (DelazJ) [inline source/site/getinvolved/development/index.rst:338]: From what i've experienced in contributing to qgis website and doc repo, file extension is not added in the TOC. Maybe it shouldn't... (no real knowledge about rst)
The "git" file neither appears in the TOC 
A question: is these chapters a part of qgisdevelopersguide or are they supposed to be aside? 

Whatever, I think there's a real issue with the TOC in http://qgis.org/fr/site/getinvolved/development/index.html and will try this week-end to propose a fix (#294)

Comment (m-kuhn) [inline source/site/getinvolved/development/index.rst:338]: I see these as parts (chapters) of the developers guide. They should be on separate pages. The dev guide is at the moment a monolithic giant piece of text that I doubt is actually read by people. I expect to increase it's impact by splitting it up.

I think #294 affects how the the TOC is specified, so I'll wait for your decision there before doing any updates.

**Comment by DelazJ** [comment]
I end up with two PRs #302 and #303, ignoring how can be the loss of translated sections (#294 was more than just a TOC wrongly displayed).
you'll also remark that you'll need to mix the content of your PR with the already existing developers guide given that some aspects are already covered.

**Comment by rduivenvoorde** [comment]
@m-kuhn @DelazJ what about merging all current PR's and clean up after that?

It is (for me) pretty hard to check the PR's one by one...

**Comment by m-kuhn** [comment]
I think there's a decision waiting for #302 vs #303 . I'd vote for the modularized approach (#302). Then we need a feedback by @rduivenvoorde concerning risk of losing translations. Then we can proceed with this one.

**Comment by rduivenvoorde** [comment]
@m-kuhn what is the status of this one? Should I be able to merge this/fix conflicts?  Or is it too messy currently and is it better if you try yourself?

**Comment by rduivenvoorde** [comment]
ah, looks like it is just the toc. Let me fix it

**Comment by m-kuhn** [comment]
This duplicates some of the doc from the developers guide. So it may be worth cleaning up a bit now that #302 was merged.

**Comment by rduivenvoorde** [comment]
@m-kuhn mmm, sorry, I end up with two different versions of qtcreator.rst. one in development, and one in qgsidevelopersguide. Can you please have a look? It's not clear for me which one is the most uptodate now.

**Comment by DelazJ** [comment]
The one in development should be removed. @m-kuhn has to move its contents into the file in developer's guide

**Comment by m-kuhn** [comment]
I'll clean and merge when I find time

**Comment by rduivenvoorde** [comment]
Thanks @m-kuhn. Yes yes, no worries, didn't mean to stress you :-)

**Comment by rduivenvoorde** [comment]
closing some stale PR's

**Pull request closed without merging**